### ISSUE TITLE
Add workaround for https://bugs.chromium.org/p/chromium/issues/detail?id=862325 ?

### ISSUE THREAD

**Issue by thehunmonkgroup** [webrtcHacks/adapter#853]
- [x] I have provided steps to reproduce (e.g. a link to a [jsfiddle](https://jsfiddle.net/))
- [x] I have provided browser name, version and adapter.js version
- [x] This issue only happens when adapter.js is used

**Note: If the checkboxes above are not checked (which you do after the issue is posted), the issue will be closed.**

**Browser name including version (e.g. Chrome 64.0.3282.119)**

Chrome 65-68

## Description

Just wondering if it would be a good idea to provide a workaround for  https://bugs.chromium.org/p/chromium/issues/detail?id=862325 ? Doesn't look like it will land for end users until 69 becomes stable. The issue affects some Android Chrome users on their front camera when a frameRate.min video constraint is specified.

I don't have a working example any more, but the problem is discussed at length in the issue filed with Chromium.

**Comment by dagingaa** [comment]
I'm not sure this is a good idea. I get where the issue is coming from, but I I don't think this should be adapter's responsibility.

adapter is a shim, normalizing the differences between browsers in implementing the WebRTC specification. This bug is not related to the specification as such, it just fails unexpectedly. 

The suggested fix here would be to ignore the constraint in certain Chrome versions, but that is setting ourselves up for failure. There is no good feature test for this, so we would have to rely on user agent sniffing to implement it. This is very error prone and could have unintended consequences.

It also fails one of the constraints of this bug: "the issue only happens when adapter.js is used". This is not the case, as this is a browser bug and doesn't have anything to do with adapter.js (not a bug in adapter.js but a bug in Chrome for Android).

Let's take another example:
WebRTC in Safari was completely broken previous to 604.4 in certain conditions (it hard crashed iOS devices), should adapter decide that WebRTC isn't supported on Safari due to that? Clearly a bug, but it would be wrong for adapter to refuse to use WebRTC as testing in the wild would make it much harder.

Shimming and workarounds for bugs is a very fine line, but in this case I don't think it should be adapter's responsibility to provide a workaround. This is not a spec inconsistency or legacy in browsers, it's a browser bug.

In summary: let adapter be a shim, it's what it's good at, and create workarounds for bugs in your own code.

**Comment by thehunmonkgroup** [comment]
Sounds reasonable, closing
